hudi-client/hudi-spark-client/src/main/scala/org/apache/hudi/HoodieSparkUtils.scala

```diff
@@ -272,4 +278,27 @@ object HoodieSparkUtils extends SparkAdapterSupport with SparkVersionsSupport wi
       validatePartitionValues = shouldValidatePartitionCols
     ).toSeq(partitionSchema)
   }
+
+  def splitHiveSlashPartitions(partitionFragments: Array[String], nPartitions: Int, sep: String, hive: Boolean): Array[String] = {
```

Comment:
preferably move it to an appropriate util in hudi-common. There is nothin spark specific in this method.